collectors/python.d.plugin/beanstalk/metadata.yaml

```diff
@@ -0,0 +1,207 @@
+name: beanstalk
```

Comment:
will there ever be need for a `name` and an `id` or should `name` be enough to act like an id?

Reply:
I think name will act as id, so for proc, name would be the individual modules

```
name: /proc/stat
...
name: /proc/sys/kernel/random/entropy_avail
```
that will help us then construct the name, this name is how it comes right out of the CSV files.

Reply:
this is exactly what i'm wondering - how would that then link to blog posts? 

`/proc/sys/kernel/random/entropy_avail`

needs to end up being able to be tagged in blog and linked something like:

https://blog.netdata.cloud/tags/entropy/

so we need, i think, some sort of guid (ideally human readable) that will be a solid key (not special chars, just text if possible) across all places.

so thinking how can we be as flexible as possible here.

Reply:
Hmm, I see, if it appears in tags, it should probably not be proc.plugin.entropy_avail..

From the script I am generating this I don't really have much to work with, I have the columns the CSV has.

So if we want something like "entropy", "statistics" and stuff, we might need to manually add it upon filling in the yamls :thinking:, what do you think?

Another idea would be to make the tags in the blogs like the last /../../../X element in the path, or even leave the tag as "entropy" and then search with regex if it exists in that last module bit :thinking: